src/main/java/com/radixdlt/crypto/encryption/ECIES.java

```diff
@@ -173,6 +173,6 @@ private ECIES() {
 	}
 
 	private static byte[] hash(byte[] data) {
-		return Hash.hash512(data);
```

Comment:
Perhaps it worth to take a look if it's possible to avoid returning raw byte array.

Reply:
I kept those methods returning byte[] in few places that do some low-level operations (Encryptor, ECIES, ECKeyPair) for performance reasons. I guess a reasonable compromise would be to not expose raw byte[] for any public methods, but keep it within the crypto package. wdyt?

Reply:
Yes, agree with making low-level ops package local.

Reply:
I've made it package local where possible, but some of the methods are used cross-package (f.e. `crypto.encryption` and `crypto`) - I've kept the low-level methods as byte[] even though they're public.